src/main/java/org/folio/rmapi/model/Identifier.java

```diff
@@ -14,10 +14,6 @@
   @JsonProperty("id")
   public String id;
 
-  @JsonInclude(JsonInclude.Include.NON_NULL)
-  @JsonProperty("source")
-  public String source;
-
```

Comment:
Just curious why this was here and now removed

Reply:
Identifier had source field just to map all fields that are returned from RM API, 
we sometimes get Identifiers from kb-ebsco-java model (which doesn't have source), and sometimes from RM API(which has source), 
to make it more consistent I removed this field